app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="form-container">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+<div class="links">
+  <%= render "devise/shared/links" %>
+</div>
+</div>
```

Comment:
- Kindly ensure that your design matches the original template as much as possible, which includes:
   - [ ] Changing the navbar background color.
   - [ ] Adding a back button (left arrow) to the left side of the navbar which functions as a back button. (This persists through the app)
   - [ ] Changing the position of the form submission button to be on the right side of the navbar. (This goes for all of the forms in the app)
   - [ ] Editing the styles of the form inputs to match the original design and have the same placeholders.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/Nasirkhan294/penny_planner/assets/62290545/bccb8c78-c1c5-49eb-bdf1-d084dc3f58ef) | ![image](https://github.com/Nasirkhan294/penny_planner/assets/62290545/d8234ba1-053f-4899-bf83-cec67e3d8001) |